### ISSUE TITLE
Update LMH login page with new logo and name [LMH]

### ISSUE THREAD

**Issue by rowo** [OpenSRP/opensrp-client-chw#1092]
Waiting on assets

**Comment by rowo** [comment]
In progress. LMH inconclusively says same logo as UNICEF app and "CBIS Demo" as the text. But this is not agreed on, so this issue remains blocked.

**Comment by rowo** [comment]
![lmh-logo](https://user-images.githubusercontent.com/1584163/73509524-68826280-43ad-11ea-9ecd-bd855190c175.png)

There was a request to remove the UNICEF portion, but it came very late. This is low priority over the client QA of the sick child form

**Comment by rowo** [comment]
They do want this.

**Comment by rowo** [comment]
@rkodev sorry also they want the app name to be "MOH VTS"

**Comment by rowo** [comment]
This looks updated:

![Screenshot_20200227-145456](https://user-images.githubusercontent.com/1584163/75481603-4f9bab80-5971-11ea-9388-fd18b4444577.png)
